.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,9 @@
+# Related Issue
+
```

Comment:
Add some text explaining what this should be (single line or example is fine)

Reply:
Updated